stellar-contract-env-host/Cargo.toml

```diff
@@ -22,18 +22,30 @@ num-integer = "0.1.45"
 
 [dev-dependencies]
 assert_matches = "1.5.0"
+optimization = "0.2.0"
+env_logger = "0.9.0"
+log = "0.4.17"
 
 [features]
 vm = ["wasmi", "parity-wasm", "stellar-contract-env-common/vm"]
+default = ["vm"]
```

Comment:
👀 Can we leave this off? I think the day we make this default, we should get rid of the feature.

Reply:
Oh, yeah, that came along by mistake. Nice catch.

Reply:
fixed.